src/Sulu/Bundle/ContentBundle/Form/Type/PageDocumentType.php

```diff
@@ -63,15 +63,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     /**
      * {@inheritdoc}
      */
-    public function setDefaultOptions(OptionsResolverInterface $options)
+    public function configureOptions(OptionsResolver $options)
```

Comment:
do we use any other form-types? can you check if there are other calls like this?